Silicon/Intel/IntelSiliconPkg/Include/IndustryStandard/FirmwareInterfaceTable.h

```diff
@@ -26,17 +30,44 @@
 #define FIT_TYPE_0C_BOOT_POLICY_MANIFEST    0x0C
 #define FIT_TYPE_0D_FSP_BOOT_MANIFEST       0x0D
 #define FIT_TYPE_10_CSE_SECURE_BOOT         0x10
+#define FIT_TYPE_1A_VAB_PROVISIONING_TABLE  0x1A
+#define FIT_TYPE_1B_VAB_KEY_MANIFEST        0x1B
+#define FIT_TYPE_1C_VAB_IMAGE_MANIFEST      0x1C
+#define FIT_TYPE_1D_VAB_IMAGE_HASH_DESC     0x1D
+#define FIT_TYPE_2C_SACM_DEBUG              0x2C
 #define FIT_TYPE_2D_TXTSX_POLICY            0x2D
+#define FIT_TYPE_2E_GRANULAR_SCRTM_ERROR    0x2E
 #define FIT_TYPE_2F_JMP_DEBUG_POLICY        0x2F
 #define FIT_TYPE_7F_SKIP                    0x7F
 
-#define FIT_POINTER_ADDRESS                 0xFFFFFFC0 ///< Fixed address at 4G - 40h
+//
+// FIT pointer definitions.
+//
+#define ADDRESS_4GB  0x0000000100000000ULL

```

Comment:
This is not needed. MdePkg/Base.h contains marco BASE_4GB

Reply:
Removed this definition and replaced as ```BASE_4GB```.